oper8/managed_object.py

```diff
@@ -10,7 +11,7 @@
 class ManagedObject:  # pylint: disable=too-many-instance-attributes
     """Basic struct to represent a managed kubernetes object"""
 
-    def __init__(self, definition):
+    def __init__(self, definition, verify_function: Callable = None):
```

Comment:
Same question about `Callable` here. Also, if you do change it, maybe throw a type hint on `definition` too!